src/components/Article.js

```diff
@@ -5,7 +5,7 @@ import Seo from "./strapi/seo";
 
 import { Typography } from "@originprotocol/origin-storybook";
 
-import Image from "next/image";
+import Image from "next/future/image";
```

Comment:
From [Next.js docs](https://nextjs.org/docs/upgrading#v13-summary), it seems like `next/future/image` is same as `next/image` from v13, any reason to do this here?

Reply:
we're on v12 i believe, it's still considered an experimental feature here